api-controllers/ServiceFabrikAdminController.js

```diff
@@ -224,9 +219,10 @@ class ServiceFabrikAdminController extends FabrikBaseController {
   getDeployment(req, res) {
     const deploymentName = req.params.name;
     const plan = catalog.getPlan(req.query.plan_id);
-    // TODO: Director Service should be used
-    this.createManager(req.query.plan_id)
-      .then(manager =>
+    Promise.try(() => DirectorService.createInstance(this.getInstanceId(deploymentName), {
+        plan_id: plan.id
+      }))
+      .then(serviceInstance =>
```

Comment:
naming this as `serviceInstance` somehow doesnt seem right. just keep it `service` or `directorService`. We dont add the suffix `instance` to all classes generally we create, so adding that specifically here simply creates even more confusion with the commonly used term `ServiceInstance`